jobs/stream.yaml

```diff
@@ -6,11 +6,21 @@ nr_threads:
   - 100%
   - 50%
   - 25%
+  - 12.5%
+  - 6.25%
+
+iterations:
+  - 10x
 
 stream:
   array_size:
     - 10000000 # 0.2GB
     - 50000000 # 1GB
+    - 128000000 # 3GB
+    - 192000000 # 4.5GB
   omp:
     - true
     - false
+  loop:
+    - 10
+    - 100
```

Comment:
prefer to only have 100 here, which is default value in original script. If you need further customize the testing like loop 10 or nr_thread, you can consider to maintain your own jobs.

Reply:
The changes are done according to the review comments.